src/Controller/ApplenewsPreviewController.php

```diff
@@ -6,11 +6,15 @@
 use Drupal\Core\Link;
 use Drupal\Core\Url;
 use Drupal\node\NodeInterface;
-use Symfony\Component\HttpFoundation\BinaryFileResponse;
-use Symfony\Component\HttpFoundation\ResponseHeaderBag;
 
+/**
+ *
+ */
 class ApplenewsPreviewController extends ControllerBase {
 
+  /**
+   *
```

Comment:
This one, and the following ones, too.

Reply:
@juampynr killed all the occurrences of '\/\*\*\n\s\*\n' in Directory modules/contrib/applenews 👍